lib/buildsystems/autotools.rb

```diff
@@ -19,7 +19,7 @@ def self.build
       end
       abort 'configure script not found!'.lightred unless File.file?('configure')
       FileUtils.chmod('+x', 'configure')
-      if system('grep -q /usr/bin/file configure')
+      if File.executable? "#{CREW_PREFIX}/bin/file"
         puts 'Using filefix.'.orange
```

Comment:
Shouldn't we only report we are running filefix if `/usr/bin/file` is actually in the configure script?

Sure checking to see if a local copy of `file` exists first too is a good idea, but that check can come afterwards only if we actually need to run filefix...

Reply:
I think it should be fine?  It aborts if configure is not found 2 lines above that.